joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -100,12 +100,17 @@ JointTrajectoryController::command_interface_configuration() const
       conf.names.push_back(joint_name + "/" + interface_type);
     }
   }
+  if (!params_.speed_scaling_command_interface_name.empty())
+  {
+    conf.names.push_back(params_.speed_scaling_command_interface_name);
```

Comment:
See my comment above. I think this does not handle a multiarm robot case. But I think it would be fairly easy to support if we use a list instead of a single value here